yaml_test_runner/Cargo.toml

```diff
@@ -13,29 +13,29 @@ opensearch = { path = "../opensearch", features = ["experimental-apis"]}
 api_generator = { path = "./../api_generator" }
 
 anyhow = "1.0"
-base64 = "^0.21"
-clap = "~2"
+base64 = "0.21"
+clap = "2"
 itertools = "0.11.0"
 Inflector = "0.11.4"
 lazy_static = "1.4.0"
 log = "0.4.8"
 once_cell = "1.4.0"
 path-slash = "0.2"
-proc-macro2 = "~1.0"
-quote = "~1.0"
+proc-macro2 = "1.0"
+quote = "1.0"
```

Comment:
These widen from `1.0.*` to `1.*.*`, probably okay because semver, and also currently proc-macro2/quote hasn't hit `1.1` yet